pandas/_libs/groupby.pyx

```diff
@@ -560,7 +560,8 @@ def _group_var(floating[:, :] out,
                int64_t[:] counts,
                floating[:, :] values,
                const int64_t[:] labels,
-               Py_ssize_t min_count=-1):
+               Py_ssize_t min_count=-1,
+               int64_t ddof=1):
```

Comment:
Can we just accept kwargs where ddof is required rather than adding it to other signatures? This is a little confusing as any / all would never need it

Reply:
Sorry - I don't quite follow. ddof is only added to var, the function that uses it. I think perhaps you were thinking this was the signature of any/all? That appears just above the change to var in the diff.

Reply:
Ah OK - sorry misread it. Thanks for clarifying